woocommerce/woocommerce
# [WIP][Accessibility] Add filter type to the remove filter label in the active-filters block

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #52091 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Install a block-based theme.
2. On the Admin, go to Appearance > Editor > Templates > WooCommerce > Product Catalog.
3. Add the Active Filters block Inside the Product Collection block.
4. Add two Filter by attribute blocks Inside the Product Collection block, below the Active Filters block. One for colors and the other one for sizes.
![image](https://github.com/user-attachments/assets/05f99382-e4f8-4992-b3c6-62f1d413fbce)
5. Save the changes and open the Shop page on the frontend.
6. Filter the page by some color and size.
7. The filters should appear at the top.
![image](https://github.com/user-attachments/assets/ba6e7cfb-a12e-456c-ab19-7aca75a9d677)
9. Turn on the screen reader.
10. Focus on the remove filter buttons.
11. Verify screen reader announces "Remove [filter type] [filter value] filter".
    e.g. "Remove Color Blue filter"
![image](https://github.com/user-attachments/assets/33541252-eabe-4593-883a-b7f57282a683)




### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
